tests/integration/integration_test.py

```diff
@@ -1150,7 +1150,8 @@ def resolve_envs(_conf):
             _conf[k] = os.getenv(v[1:])
 
 
-test_modes = ['consumer', 'producer', 'avro', 'performance', 'admin', 'avro-https', 'avro-basic-auth', 'throttle']
+test_modes = ['consumer', 'producer', 'avro', 'performance', 'admin',
```

Comment:
Not needed

Reply:
okay will revert it it was to avoid flake8 errors